composer.json

```diff
@@ -38,7 +38,8 @@
     "phpunit/phpunit": "^6.2",
     "squizlabs/php_codesniffer": "^3.0",
     "codeception/codeception": "^2.5.3",
-    "consolidation/robo": "^1.2"
+    "consolidation/robo": "^1.2",
+    "phpstan/phpstan": "@stable"
```

Comment:
Did you look at adding [`phpstan/phpstan-phpunit`](https://github.com/phpstan/phpstan-phpunit)? Does it make static analysis of our unit tests any more feasible?

Reply:
No, I didn't try, unit tests was excluded from phpstan checking as they contain too many issues. We can add a task  about this to our backlog.